mount/mount_test.go

```diff
@@ -46,45 +48,51 @@ func TestBadDev(t *testing.T) {
-		{in: "rw,nosuid,nodev,noexec,relatime,hugetlb", out: "rw,nosuid,nodev,noexec,relatime,hugetlb"},
-		{in: "rw,nosuid,nodev,noexec,relatime,memory", out: "rw,nosuid,nodev,noexec,relatime,memory"},
-		{in: "rw,nosuid,nodev,noexec,relatime,mode=700", out: "rw,nosuid,nodev,noexec,relatime,mode=700"},
-		{in: "rw,nosuid,nodev,noexec,relatime,net_cls,net_prio", out: "rw,nosuid,nodev,noexec,relatime,net_cls,net_prio"},
-		{in: "rw,nosuid,nodev,noexec,relatime,nsdelegate", out: "rw,nosuid,nodev,noexec,relatime,nsdelegate"},
-		{in: "rw,nosuid,nodev,noexec,relatime,perf_event", out: "rw,nosuid,nodev,noexec,relatime,perf_event"},
-		{in: "rw,nosuid,nodev,noexec,relatime,pids", out: "rw,nosuid,nodev,noexec,relatime,pids"},
-		{in: "rw,nosuid,nodev,noexec,relatime,rdma", out: "rw,nosuid,nodev,noexec,relatime,rdma"},
-		{in: "rw,nosuid,nodev,noexec,relatime,size=3902136k,mode=755", out: "rw,nosuid,nodev,noexec,relatime,size=3902136k,mode=755"},
-		{in: "rw,nosuid,nodev,noexec,relatime,size=5120k", out: "rw,nosuid,nodev,noexec,relatime,size=5120k"},
-		{in: "rw,nosuid,nodev,noexec,relatime,xattr,name=systemd", out: "rw,nosuid,nodev,noexec,relatime,xattr,name=systemd"},
-		{in: "rw,nosuid,nodev,relatime,size=3902132k,nr_inodes=975533,mode=700,uid=1000,gid=1000", out: "rw,nosuid,nodev,relatime,size=3902132k,nr_inodes=975533,mode=700,uid=1000,gid=1000"},
-		{in: "rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000", out: "rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000"},
-		{in: "rw,nosuid,noexec,relatime,size=19429784k,nr_inodes=4857446,mode=755", out: "rw,nosuid,noexec,relatime,size=19429784k,nr_inodes=4857446,mode=755"},
-		{in: "rw,relatime", out: "rw,relatime"},
-		{in: "rw,relatime,fd=28,pgrp=1,timeout=0,minproto=5,maxproto=5,direct,pipe_ino=24647", out: "rw,relatime,fd=28,pgrp=1,timeout=0,minproto=5,maxproto=5,direct,pipe_ino=24647"},
-		{in: "rw,relatime,fmask=0022,dmask=0022,codepage=437,iocharset=iso8859-1,shortname=mixed,errors=remount-ro", out: "rw,relatime,fmask=0022,dmask=0022,codepage=437,iocharset=iso8859-1,shortname=mixed,errors=remount-ro"},
-		{in: "rw,relatime,pagesize=2M", out: "rw,relatime,pagesize=2M"},
-		{in: "sw", out: "sw"},
+		{in: "defaults", flag: 0, opt: ""},
+		{in: "ro,defaults", flag: unix.MS_RDONLY, opt: ""},
+		{in: "ro,nodev,relatime", flag: unix.MS_RELATIME | unix.MS_RDONLY | unix.MS_NODEV, opt: ""},
+		{in: "ro,nosuid,nodev,noexec,size=4096k,nr_inodes=1024,mode=755", flag: unix.MS_RDONLY | unix.MS_NOSUID | unix.MS_NODEV | unix.MS_NOEXEC, opt: "size=4096k,nr_inodes=1024,mode=755"},
+		{in: "rw", flag: 0, opt: ""},
+		{in: "rw,nosuid,nodev", flag: unix.MS_NOSUID | unix.MS_NODEV, opt: ""},
+		{in: "rw,nosuid,nodev,noexec,relatime", flag: unix.MS_RELATIME | unix.MS_NOSUID | unix.MS_NODEV | unix.MS_NOEXEC, opt: ""},
+		{in: "rw,nosuid,nodev,noexec,relatime", flag: unix.MS_RELATIME | unix.MS_NOSUID | unix.MS_NODEV | unix.MS_NOEXEC, opt: ""},
```

Comment:
duplicated test?  (same as prev line).  i see this one pop up a few times (L67 L68, etc).  L65 and L69 look the same too?)

Reply:
fixed, thanks.